src/librustc/session/config.rs

```diff
@@ -869,6 +881,8 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
         "disable the use of the redzone"),
     relocation_model: Option<String> = (None, parse_opt_string, [TRACKED],
          "choose the relocation model to use (rustc --print relocation-models for details)"),
+    relro_level: Option<RelroLevel> = (None, parse_relro_level, [TRACKED],
```

Comment:
Would it be an issue that the change in default behavior is insta-stable, while the new knob to control it is still unstable?  One could use `-Clink-arg=-Wl,-z,norelro` to cancel the first part, but I don't think `-z now` has an off switch.